Tiltfile

```diff
@@ -350,8 +346,20 @@ def deploy_worker_templates(template, substitutions):
     yaml = yaml.replace('"', '\\"')  # add escape character to double quotes in yaml
     flavor_name = os.path.basename(flavor)
     flavor_cmd = "RANDOM=$(bash -c 'echo $RANDOM'); CLUSTER_NAME=" + flavor.replace("windows", "win") + "-$RANDOM; make generate-flavors; echo \"" + yaml + "\" > ./.tiltbuild/" + flavor + "; cat ./.tiltbuild/" + flavor + " | " + envsubst_cmd + " | " + kubectl_cmd + " apply -f - && echo \"Cluster \'$CLUSTER_NAME\' created, don't forget to delete\""
+
+    # wait for kubeconfig to be available
+    flavor_cmd += "; until " + kubectl_cmd + " get secret ${CLUSTER_NAME}-kubeconfig > /dev/null 2>&1; do sleep 5; done; " + kubectl_cmd + " get secret ${CLUSTER_NAME}-kubeconfig -o jsonpath={.data.value} | base64 --decode > ./${CLUSTER_NAME}.kubeconfig; chmod 600 ./${CLUSTER_NAME}.kubeconfig; until " + kubectl_cmd + " --kubeconfig=./${CLUSTER_NAME}.kubeconfig get nodes > /dev/null 2>&1; do sleep 5; done"
+
+    # install calico
+    if "ipv6" in flavor_name:
+        calico_values = "./templates/addons/calico-ipv6/values.yaml"
+    elif "dual-stack" in flavor_name:
+        calico_values = "./templates/addons/calico-dual-stack/values.yaml"
+    else:
+        calico_values = "./templates/addons/calico/values.yaml"
+    flavor_cmd += "; " + helm_cmd + " repo add projectcalico https://projectcalico.docs.tigera.io/charts; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install calico projectcalico/tigera-operator -f " + calico_values + " --namespace tigera-operator --create-namespace; kubectl --kubeconfig ./${CLUSTER_NAME}.kubeconfig apply -f ./templates/addons/calico/felix-override.yaml"
```

Comment:
should we wait for some outcome from the helm install (like the existence of a CRD, for example) before applying the override? I assume there is some serialization required there?

Reply:
that's a good point, from my observations as soon as the Helm chart is applied the CRD exists but let me make sure there isn't a possible race

Reply:
Thinking about this more... I think the helm CLI exits success when the apiserver has admitted the full set of resources. Because the CRDs we're talking about don't have any interesting state transitions, we should be able to reliably update `default` of type `FelixConfiguration` w/ `spec.featureDetectOverride: "ChecksumOffloadBroken=true"`.

But we should be aware that we don't fully understand all the side-effects that the set of k8s components that calico is installing via helm. It's _possible_ there's some conflicting update to that same CRD resource that happens during "calico bootstrap", so if we see weird results in our E2E tests we can check here.

Reply:
Agreed. Also not that this workaround can be reverted as soon as Calico v1.25 comes out, the fix has already been merged, just waiting for the release